.github/workflows/ci.yml

```diff
@@ -236,11 +232,14 @@ jobs:
             which pip
 
       - name: Compilation Cache
-        uses: hendrikmuhs/ccache-action@v1.2
+        uses: actions/cache@v3
+        id: compilation-cache
         with:
-          variant: ${{ startsWith(runner.os, 'windows') && 'sccache' || 'ccache' }}  # fake ternary
```

Comment:
`variant` is for CI runs with msys instead of regular windows. But I don't see any clang-cl CI runs. Am I missing something? The newer action does not have a `variant` key.